docs/src/modules/components/BackToTop.js

```diff
@@ -1,56 +1,48 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 import useScrollTrigger from '@mui/material/useScrollTrigger';
 import Box from '@mui/material/Box';
 import Fab from '@mui/material/Fab';
 import Tooltip from '@mui/material/Tooltip';
 import KeyboardArrowUpRoundedIcon from '@mui/icons-material/KeyboardArrowUpRounded';
-import Zoom from '@mui/material/Zoom';
+import Fade from '@mui/material/Fade';
```

Comment:
Replace the Zoom with a Fade animation to fix the broken tooltip animation and to reduce the visual distraction when scrolling